tests/unit/python/fledge/services/core/api/test_certificate_store.py

```diff
@@ -244,7 +244,6 @@ async def test_bad_delete_cert_with_invalid_filename(self, client, cert_name, ac
     
     @pytest.mark.parametrize("cert_name, actual_code, actual_reason", [
         ('root.txt', 400, "Accepted file extensions are ('.cert', '.cer', '.crt', '.json', '.key', '.pem')"),
-        ('admin.key', 400, "Admin certs cannot be deleted"),
```

Comment:
key canbe deleted.

admin once copied/ pulled from foglamp can also delete the admin (.cert) certs from foglamp, but disallow to normal user and when auth is optional.